src/sherpa_ai/database/user_usage_tracker.py

```diff
@@ -3,16 +3,16 @@
 import boto3
 from anyio import Path
 from sqlalchemy import TIMESTAMP, Boolean, Column, Integer, String, create_engine
-from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import sessionmaker
```

Comment:
can you add declartive_base to the import from sqlalchemy.orm like bellow`from sqlalchemy.orm import sessionmaker,declartive_base `